x-pack/plugins/lens/public/app_plugin/app.test.tsx

```diff
@@ -486,30 +475,6 @@ describe('Lens App', () => {
         expect(getButton(instance).disableButton).toEqual(true);
       });
 
-      it('shows a disabled save button when there are no changes to the document', async () => {
```

Comment:
Just to confirm - was this test removed because we don't even do this (disabling the save button when there are no changes)?

Reply:
That's right. This test wasn't actually correct, and the tested behavior is identical to the test right after it.